src/main/java/io/jaegertracing/tests/CreateSpansRunnable.java

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2018 The Jaeger Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.jaegertracing.tests;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import io.jaegertracing.internal.JaegerTracer;
+import io.jaegertracing.tests.model.TestConfig;
+import io.opentracing.Span;
+import io.opentracing.tag.Tags;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @author Pavol Loffay
+ */
+@Slf4j
+public class CreateSpansRunnable implements Runnable {
+    class SpansTimer extends TimerTask {
+
+        private long delay = 0;
+
+        public SpansTimer() {
```

Comment:
Not sure why timmer name it seems it only report spans.

Reply:
It's cleaner to have one simple runnable which just reports spans and then reuse it in a new class running as a timer.

Reply:
I will change the class name. I use this class as runnable if we run tests in `Long` running mode. For the quick test, I call `.run()` method.